### PR TITLE
Move assigned projects outside of form

### PR DESCRIPTION
**Pull request by cproensa**
Move the list of assigned projects outside of the form in account_page.
This list is not editable, so it must not be part of the form.

Fixes: #21552

seleccion_151

### CODE REVIEW

File: account_page.php [hunk 1]
```diff
@@ -294,6 +273,48 @@
...
+<?php
+$t_projects = user_get_assigned_projects( auth_get_current_user_id() );
+if( count( $t_projects ) > 0 ) {
+?>
+	<div class="space-10"></div>
+
+	<div class="widget-box widget-color-blue2">
+		<div class="widget-header widget-header-small">
+			<h4 class="widget-title lighter">
+				<i class="ace-icon fa fa-user"></i>
```
Comment (syncguru) [inline account_page.php]: Should we use another icon for this section? Maybe the same one we use in manage projects page: http://fontawesome.io/icon/puzzle-piece/

**Comment by cproensa** [comment]
> Should we use another icon for this section? Maybe the same one we use in manage projects

yes, makes sense.
I updated my-account page and manage-user page.

**Comment by vboctor** [comment]
@cproensa 

- How is the assigned projects field different from other read-only fields on the form?
- Does this change also impact the manage user edit page?
- Maybe it would be good to align the table columns for the new table with the one above it for user information.

**Comment by cproensa** [comment]
> How is the assigned projects field different from other read-only fields on the form?

The main issue is that this item is variable in size. The form button can be outside screen when a long enough list of project exists.
By moving this, the form remains in constant size.

> Does this change also impact the manage user edit page?

Only change is the widget icon as suggested by Rafik

> Maybe it would be good to align the table columns for the new table with the one above it for user information.

Afaik, we dont have any standards yet on managing columns sizes in this kind of table-forms. That would be a good point to be worked on, since I feel that on wide screens the widgets and spacings can get too wide-spread.

File: account_page.php [hunk 2]
```diff
@@ -294,6 +273,48 @@
 	</div>
 </div>
 
+<?php
+$t_projects = user_get_assigned_projects( auth_get_current_user_id() );
+if( count( $t_projects ) > 0 ) {
```
Comment (vboctor) [inline account_page.php]: !empty() is more efficient

File: account_page.php [hunk 3]
```diff
@@ -294,6 +273,48 @@
...
+		</div>
+		<div class="widget-body">
+			<div class="widget-main no-padding">
+				<div class="table-responsive">
+					<table class="table table-bordered table-condensed table-striped">
+						<?php
+							echo '<tr>';
+							echo '<td class="category">' . lang_get( 'assigned_projects' ) . '</td>';
+							echo '<td>';
+							foreach( $t_projects AS $t_project_id=>$t_project ) {
```
Comment (vboctor) [inline account_page.php]: Add spacing around `=>`

**Comment by vboctor** [comment]
We can have re-usable code that prints out a section with a set of projects.  It would take the section title and array of projects/project ids as input.

**Comment by cproensa** [comment]
seleccion_154

by the way, updated header styles for manage projects table

seleccion_155

> We can have re-usable code that prints out a section with a set of projects

Where would you reuse this section, as is?

**Comment by vboctor** [comment]
> Where would you reuse this section, as is?

I was thinking we can potentially re-use for account page, manage projects page and manage user page.

**Comment by cproensa** [comment]
>  re-use for account page, manage projects page and manage user page

in my opinion, won't fit...
- manage project page is more complex, as it features sorting. And doesnt show access level (does not make sense there)
- manage user page is close, but there the table is actually a form.

Anyway, the code for this table is very concise. Not a big problem to keep it ad-hoc.

**Pull request merged**